go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/devfile/library
 go 1.13
 
 require (
-	github.com/devfile/api/v2 v2.0.0-20210223145532-81859eaef987
+	github.com/devfile/api/v2 v2.0.0-20210304212617-bfc3f501616b
```

Comment:
if you're updating devfile/api, schema needs to be updated too

This is automated now, just run the script as the README suggest https://github.com/devfile/library/blob/master/README.md

Reply:
I ran the script again "./updateApi.sh 2.0.0":

martinmulholland@mmulholl-mac library % ./updateApi.sh 2.0.0
Updating devfile/api in go.mod
Get latest schema
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  160k  100  160k    0     0   540k      0 --:--:-- --:--:-- --:--:--  540k
Write to go file
Writing to file: ./pkg/devfile/parser/data/v2/2.0.0/devfileJsonSchema200.go

but git status does not list pkg/devfile/parser/data/v2/2.0.0/devfileJsonSchema200.go as having changed.

Reply:
we're updating 2.1.0 not 2.0.0 but i tried it out and there were no changes to the schema